config/initializers/rails_admin.rb

```diff
@@ -74,9 +113,35 @@
           "#{self.name}_field sponsorship_fields_toggler"
         end
       end
+      field :name
+      field :description
+      field :point_value
+      field :achievement_name
+      field :state
+      field :defense_period
+      field :defense_points
+      field :unsolved_increment_period
+      field :unsolved_increment_points
+      field :initial_shares
+      field :solved_decrement_shares
+      field :first_capture_point_bonus
+      field :design_phase
+      field :sponsored
+      field :sponsor
+      field :sponsor_logo
+      field :sponsor_description
+      field :game
+      field :category_ids
+      field :defense_flags
```

Comment:
This is nearly every field (except ~4) on the game model, can we do this as a blacklist instead of a whitelist?

2nd example: https://github.com/sferik/rails_admin/wiki/Fields#visibility-and-ordering